### ISSUE TITLE
Current Location only displays results when Input is in focus

### ISSUE THREAD

**Issue by Aryk** [FaridSafi/react-native-google-places-autocomplete#496]
Clicking on the current Location "blurs" the component which set listViewDisplayed:false.

The results get "loaded", but not displayed until you re-focus the input component.

I'm not sure if this was intentional but it basically means nothing will happen unless the user clicks on the input box again, triggering the _onFocus function again.

I'm trying to find a hack around this:

```js
    } else if (rowData.isCurrentLocation === true) {
      // display loader
      this._enableRowLoader(rowData);

      this.setState({
        text: this._renderDescription( rowData ),
      });

      // this.triggerBlur(); // hide keyboard but not the results
      delete rowData.isLoading;
      this.getCurrentLocation();

    } else {
```

I commented out `// this.triggerBlur(); // hide keyboard but not the results` and it works now as I expect it to.

Why was the blur added and is this the expected behavior?

**Comment by Aryk** [comment]
I believe you need to change:

```js
             this.setState({
                dataSource: this.buildRowsFromResults(results),
              });
```

to 

```js
             this.setState({
                dataSource: this.buildRowsFromResults(results),
                listViewDisplayed: true,
              });
```

on `_requestNearby`

**Comment by harishsn** [comment]
Even i am facing the same issue..

**Comment by luismasg** [comment]
So i had the same issue. I even copied the same code from the current github repo.
Strange .
Its really nice to see people contributing and updating it

**Comment by tinBilicCoreline** [comment]
I have the same issue and when I comment out what Aryk said it works, hope they will resolve this soon.

**Comment by tinBilicCoreline** [comment]
textInputProps={{ onBlur: _.noop() }}

whereas _ is from lodash

this solved this problem for me

**Comment by carylaw** [comment]
my **Current Location** solution

```
<GooglePlacesAutocomplete
  currentLocation={true} // Will add a 'Current location' button at the top of the predefined places list
  currentLocationLabel="Current Location"
  nearbyPlacesAPI="GoogleReverseGeocoding"
  textInputProps={{ onBlur: () => {} }}
..../>
```

**Comment by Sbarcenas** [comment]
>   currentLocation={true} // Will add a 'Curr

This works to me

**Comment by bell-steven** [comment]
fixed in 1.7.0.
